vertx-sql-client/src/main/java/io/vertx/sqlclient/Transaction.java

```diff
@@ -51,8 +51,8 @@
    * Return the transaction completion {@code Future} that
    *
    * <ul>
-   *   <li>succeeds when the transaction commits</li>
-   *   <li>fails with {@link TransactionRollbackException} when the transaction rollbacks</li>
+   *   <li>succeeds when the transaction commits and</li>
```

Comment:
why is _and_ added ? it's a bullet list

Reply:
"and" is missing in the method summary of Transaction.completion on
https://vertx-web-site.github.io/docs/apidocs/io/vertx/sqlclient/Transaction.html because all browsers that I've tested don't show any bullet list: Firefox, Opera, Chrome, Safari
![transaction-bullet-point](https://user-images.githubusercontent.com/533612/99684559-e2d9bb80-2a81-11eb-9667-537a7b0e7613.png)

Reply:
ok, then I think best is that you also remove the UL/LI markup in your PR

Reply:
Removed.

Reply:
thanks